### PR TITLE
Create a secret route

### PR DESCRIPTION
**Pull request by Haim-S**
I added an isProtected key to the array of paths, which checks which path needs protection.

And then, I built in the route tag, a condition that checks if it's protected, then it doesn't go through, and if so, then it goes through.

Hope you liked it, this is my first open source contribution.

### CODE REVIEW

File: src/routes/ProtectedRoute.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Navigate } from 'react-router-dom';
+
+const ProtectedRoute = ({children}:any) => {
```
Comment (ArkadiK94) [inline src/routes/ProtectedRoute.tsx:4]: Hmm, we don't have authentication on the site so we don't need a Protected Route, the option to get to this page will be through "hidden secret egg" as the storybook.

**Review by ArkadiK94 — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution. I have some comments. The main one is to use yarn instead of npm.

**Comment by ArkadiK94** [comment]
@NoamGaash can we close this pr?

@Haim-S your contribution is very appreciated even though it wasn't as requested because some misunderstanding. You are welcome to take another issue :)

**Comment by NoamGaash** [comment]
@Haim-S @ArkadiK94 it's inactive for a couple of weeks now, I believe it can be closed.
@Haim-S please let us know if you want to reopen it, maybe there's a misunderstanding from our side,
thank you for participating openbus development efforts

**Comment by Haim-S** [comment]
Thank you for the opportunity to contribute to your open source, I would
really like to understand where I can contribute and what you would really
like, if there is a meeting on Discord then I would love to join and keep
me updated, thanks.

‫בתאריך שבת, 16 בדצמ׳ 2023 ב-11:52 מאת ‪NoamGaash‬‏ <‪
***@***.***‬‏>:‬

**Comment by NoamGaash** [comment]
Hi @Haim-S , thank you!
Feel free to join our [discord](https://discord.com/channels/1089589164707684443/1150649893799608401) channel and [slack](https://join.slack.com/t/hasadna/shared_invite/zt-21qipktl1-7yF4FYJVxAqXl0wE4DlMKQ) channel.
Also, consider making smaller pull requests - so the development cycle will be shorter. You don't have to solve entire issue in a single pull request - any improvement is a great thing.
how about solving this one?
https://github.com/hasadna/open-bus-map-search/issues/80

**Pull request closed without merging**